buildpipeline/DotNet-CoreFx-Trusted-Windows.json

```diff
@@ -176,10 +176,10 @@
         "definitionType": "task"
       },
       "inputs": {
-        "solution": "$(Build.SourcesDirectory)\\corefx\\src\\tests.builds",
+        "solution": "$(Build.SourcesDirectory)\\corefx\\src\\upload-tests.proj",
         "platform": "",
         "configuration": "",
-        "msbuildArguments": "$(PB_CreateHelixArguments) /p:\"Branch=$(SourceBranch)\" /p:\"CloudDropAccountName=$(PB_CloudDropAccountName)\" /p:\"CloudResultsAccountName=$(PB_CloudResultsAccountName)\" /p:\"CloudDropAccessToken=$(CloudDropAccessToken)\" /p:\"CloudResultsAccessToken=$(OutputCloudResultsAccessToken)\" /p:\"HelixApiAccessKey=$(HelixApiAccessKey)\" /p:\"HelixApiEndpoint=$(PB_HelixApiEndPoint)\" /p:\"OfficialBuildId=$(OfficialBuildId)\"",
```

Comment:
Good point, will update the queue list.  The queues aren't missing though, they're expressed slightly differently in all three build types (OSX, Linux, and Windows) for ... reasons.

See https://github.com/dotnet/corefx/pull/16868/files/aa9d8c7e0ffe011396cd5a7e51b2e60b2d72270a#diff-c9539d529b27144215b930ab94c59342R258 .  Will update the Windows queue list and push that.